src/main/java/com/selfxdsd/todocli/Todo.java

```diff
@@ -22,12 +22,16 @@
  */
 package com.selfxdsd.todocli;
 
+// @todo #4:60min  Add interface Todos which should represent an iterable of Todo objects.
+//  Afterwards, add class JsonTodos, which should represent the TODOs as JsonArray.
```

Comment:
@lukacupic It's better if you add todos in JavaDocs. Move this Todo to the class' JavaDoc. I'm pretty sure Checkstyle complains about it.